app/class/Optlist.php

```diff
@@ -50,6 +50,10 @@ public function listhtml(array $pagelist, Page $currentpage, Servicerender $rend
 
         $li = '';
 
+        $lang = $currentpage->lang() == '' ? Config::lang() : $currentpage->lang();
```

Comment:
what do you think of using `empty()` here ?

Reply:
`empty()` is mostly useful when the can can be unset. For instance in associative arrays. Here the value is always set. Also it is supposed to never be null based on the type hints.